muted-tests.yml

```diff
@@ -444,6 +444,9 @@ tests:
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackLookupJoinIT
   method: testLookupExplosionBigString
   issue: https://github.com/elastic/elasticsearch/issues/138510
+- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
+  method: test {csv-spec:inlinestats.MvMinMvExpand}
+  issue: https://github.com/elastic/elasticsearch/issues/137679
```

Comment:
This is tracked in the meta-issue blocking deployment of pushing functions to field loads: https://github.com/elastic/elasticsearch/issues/137679